project/Build.scala

```diff
@@ -1667,14 +1667,14 @@ object Build {
           case "2.11.12" =>
             Some(ExpectedSizes(
                 fastLink = 519000 to 520000,
-                fullLink = 107000 to 108000,
+                fullLink = 108000 to 109000,
                 fastLinkGz = 66000 to 67000,
                 fullLinkGz = 28000 to 29000,
             ))
 
           case "2.12.12" =>
             Some(ExpectedSizes(
-                fastLink = 781000 to 782000,
+                fastLink = 780000 to 781000,
```

Comment:
I think this is the first change that has different directions based on the scala version :P